Cargo.toml

```diff
@@ -191,16 +190,9 @@ forest_test_utils = { path = "./utils/test_utils" }
 forest_utils = { path = "./utils/forest_utils" }
 
 [profile.dev]
-debug = 0
```

Comment:
why wouldn't we want debug info in a dev build?

Reply:
@aatifsyed https://github.com/ChainSafe/forest/pull/2918

Long story short, a `make test-all` would bloat to 71G (or more) with this. Given the overhead and how seldom one needs to go full GDB, it makes sense to disable it. Do you agree?